plugins/linux-pipewire/pipewire.h

```diff
@@ -34,9 +33,18 @@ struct obs_pipwire_connect_stream_info {
 	struct {
 		bool cursor_visible;
 	} screencast;
+	struct {
+		const struct spa_rectangle *resolution;
+		const struct spa_fraction *framerate;
+	} video;
```

Comment:
This member does not look to be used in screencast code, is it okay to have `video` as name ?
The actual name makes it sound like it could be used by it.

Reply:
It can eventually be used by the screencasting code, for example, to renegotiate the framerate when it's changed in OBS. So I think it's fine. I specifically called it `video` because it could be shared with screenshare or other potential features that don't exist yet, like virtual monitors, or p2p media content sharing.